packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -95,6 +96,13 @@ interface PseudoStylesProps {
   _notFirst?: BaseBoxProps;
   _notLast?: BaseBoxProps;
   _placeholder?: BaseBoxProps;
+  _child?: BaseBoxProps;
```

Comment:
Same here, I don't _think_ we need them.

Reply:
Will this come into play when someone wants to create custom radio buttons, using RadioGroup?

Reply:
We should just export the sibling box at that stage as it's own component, potentially in the box package or something

Reply:
Removed and moved to `SiblingBox`